README.md

```diff
@@ -347,7 +347,7 @@ options:
 A simple example of inference is shown below:
 
 ```bash
- runai exec_gpu python bin/inference.py --model_path /scratch/shared/py4cast/logs/camp0/poesy/halfunet/sezn_run_dev_30 --date 2021061621 --dataset poesy_infer --infer_steps 2
+ runai exec_gpu python -m pdb bin/inference.py --model_path /scratch/shared/py4cast/logs/camp0/poesy/halfunet/sezn_run_dev_12 --date 2021061621 --dataset poesy_infer --infer_steps 3
```

Comment:
is it needed to change this command in the readme ?

Reply:
Yes I have deleted all my old poesy runs, there were too many.